apps/web/src/middleware.ts

```diff
@@ -6,6 +6,7 @@ const middleware: NextMiddleware = (request) => {
   const requestHost = request.headers.get('host')
   const baseUrl = process.env.BASE_URL
 
+  // FIXME: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security states "Note: This is more secure than simply configuring a HTTP to HTTPS (301) redirect on your server, where the initial HTTP connection is still vulnerable to a man-in-the-middle attack.". But they keep applying this redirect in recommended SSL configs: https://ssl-config.mozilla.org/
```

Comment:
@hugues-m 

> Je n'ai pas compris, 'Strict-Transport-Security' renvoie une erreur en http au lieu d'un redirect ?
> Tu peux en tout cas laisser la doc c'est interessant :)
> J'attend de voir le comportement lorsque cette branche pourra être déployée

J'ai plutôt compris que `Strict-Transport-Security` informe le navigateur qu'il faut accéder au site en HTTPS. Si le navigateur a déjà accédé au site en HTTPS (il connaît le certificat), il bascule alors automatiquement de HTTP vers HTTPS. C'est ce que je comprends de la section [How the browser handles it](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security#how_the_browser_handles_it) 👀